### DISCUSSION TITLE
Branch structure and names

### DISCUSSION THREAD

**Discussion by zlaic** [Zeinab-HasasZiabari/Tafe24S1T102_Vir#5] [Q&A] [answered]
How should we structure the our branches, are there naming conventions for branches that we should adhere to?

**Comment by Zeinab-HasasZiabari** [comment, marked as answer]
In GitHub, we have "main" branch and development branch. 
We will create feature branches "feature_mortgage" for the mortgage calculator feature and 
"feature_currency" for the currency converter feature from "development" branch. 
The structure will be : 

![branch structure](https://github.com/Zeinab-HasasZiabari/Tafe24S1T102_Vir/assets/161718678/fdd42d77-e0ea-4e4a-8b52-4f2d100ee3f7)
